tests/integ/sagemaker/jumpstart/conftest.py

```diff
@@ -113,6 +133,37 @@ def _teardown():
     bucket = s3_resource.Bucket(test_cache_bucket)
     bucket.objects.filter(Prefix=test_suite_id + "/").delete()
 
+    # delete private hubs
+    _delete_hubs(sagemaker_session)
+
+
+def _delete_hubs(sagemaker_session):
+    # list Hubs created by PySDK integration tests
+    list_hub_response = sagemaker_session.list_hubs(
+        name_contains=os.environ[ENV_VAR_JUMPSTART_SDK_TEST_HUB_NAME]
```

Comment:
can we create a utility to get the hub name from the env var?

Reply:
just to confirm you mean a function to get `os.environ[ENV_VAR_JUMPSTART_SDK_TEST_HUB_NAME]` ?

Reply:
question: would there ever be several hubs here?
The `_setup()` method seems to create only one hub, and since the name is set there can only be one, can it not?

Reply:
@JGuinegagne yeah there'll be only one, I changed this recently previously I was deleting all the hubs starting with a specific prefix but that's messing up with the concurrent pytest executions. Let me no use list_hubs here, thanks.